docs/releases/3.2.txt

```diff
@@ -95,6 +95,31 @@ or on a per-model basis::
 In anticipation of the changing default, a system check will provide a warning
 if you do not have an explicit setting for :setting:`DEFAULT_AUTO_FIELD`.
 
+.. admonition:: Migrating auto-created through tables.
+
+    The value of ``DEFAULT_AUTO_FIELD`` will be respected when creating new
+    auto-created through tables for many-to-many relationships.
+
+    The primary key of auto-created through tables cannot currently be updated
+    by the migrations framework, however.
+
+    This means that if you switch the value of ``DEFAULT_AUTO_FIELD`` and then
+    generate migrations, the primary keys of the related models will be
+    updated, as will the foreign keys from the through table, but the primary
+    key of the auto-created through table will not be migrated.
+
+    In order to address this, you should add a
+    :class:`~django.db.migrations.operations.RunSQL` operation to your
+    migrations to perform the required ``ALTER TABLE`` step. You can check the
+    existing table name through ``sqlmigrate``, ``dbshell``, or with the
+    field’s ``remote_field.through._meta.db_table`` property
+
+    Note that explicitly defined through models are already handled by the
+    migrations system.
+
+    Allowing automatic migrations for the primary key of auto-created through
+    tables :ticket:`may be implemented at a later date <32674>`.
```

Comment:
Maybe replace this with a shorter piece of text linking to the other admonition rather than copy-paste? (If for some reason this is edited in one place in the future, it may be missed in the other.)